Cargo.toml

```diff
@@ -135,6 +135,7 @@ hyper-rustls = { version = "0.27.2", default-features = false, features = [
     "logging",
 ] }
 hyper-util = { version = "0.1" }
+indexmap = "2.7"
```

Comment:
This dependency is also used in the `restate-utoipa` crate. It could become a workspace dependency there now.